antarest/launcher/service.py

```diff
@@ -228,7 +228,7 @@ def run_study(
         job_uuid = self._generate_new_id()
         logger.info(f"New study launch (study={study_uuid}, job_id={job_uuid})")
         study_info = self.study_service.get_study_information(uuid=study_uuid, params=params)
-        solver_version = study_version or study_info.version
+        solver_version = study_version or study_info.version.__format__(format_spec="ddd")
```

Comment:
I think `__format__` should not be used directly, but rather through the usual formatting syntaxes like `f"{study_info.version:ddd}"`